words.html

```diff
@@ -1200,7 +1200,7 @@
 5. Die Evolution des Misstrauens
 </p>
 <p id="chapter_noise">
-6. Feler machen
```

Comment:
This one is intentional and also present in the original. See here: https://github.com/ncase/trust/blob/gh-pages/words.html#L666 https://github.com/ncase/trust/blob/gh-pages/words.html#L1214